src/user-post/entities/Post.entity.ts

```diff
@@ -5,21 +5,35 @@ import {
   ManyToOne,
   PrimaryGeneratedColumn,
   UpdateDateColumn,
+  JoinColumn,
+  OneToMany,
 } from 'typeorm';
 import { User } from '../../authentication/entities/user.entity';
+import { Comments } from '../../user-comment/entities/Comments.entity';
 
 @Entity()
-export class Post {
+export class Posts {
```

Comment:
Why we made it plural? Post entity is singular. Let's keep consistent both are making sense